cg/server/admin.py

```diff
@@ -564,16 +564,16 @@ class DeliveryView(BaseView):
 
 
 class FamilySampleView(BaseView):
-    """Admin view for Model.FamilySample"""
+    """Admin view for Model.caseSample"""
```

Comment:
```suggestion
    """Admin view for Model.CaseSample"""
```

Reply:
I have not gotten around to renaming this model yet, will do though 👍